app/models/group.rb

```diff
@@ -355,12 +355,31 @@ def discussion_private_default
     end
   end
 
-  def add_member!(user, inviter=nil)
-    if is_parent?
-      if (memberships_count.to_i > max_size.to_i)
-        raise Group::MaximumMembershipsExceeded
-      end
+  def org_members_count
+    if is_subgroup?
+      parent.org_members_count
+    else
+      Membership.active.where(group_id: [id, subgroups.pluck(:id)].flatten).pluck(:user_id).uniq.count
     end
+  end
+
+  def approaching_max_size?
+    ENV['SHOW_LOOMIO_ORG_MARKETING'] && org_members_count > (org_max_size * 0.8)
```

Comment:
Is 'SHOW_LOOMIO_ORG_MARKETING' essentially 'this is loomio.org, instead of some 3rd party install'? If so, is it possible to change it so it's a little more clear, before we start using it everywhere?